doc/esmvalcore/recipe.rst

```diff
@@ -1,30 +1,39 @@
 .. _recipe:
 
-*****************
-ESMValTool recipe
-*****************
+******
+Recipe
+******
 
-Recipes are the instructions telling ESMValTool about the user who wrote the
-recipe, the datasets which need to be run, the preprocessors that need to be
+Overview
+========
+
+After ``config-user.yml``, the ``recipe.yml`` file the second the user inputted file
+to ESMValTool at each run time point. Rceipes contain the data and data analysis
+information and instructions needed to run the diagnostic(s), as well as specific
+diagnostic-related instructions.
+
+Broadly, recipes contain information on the user who wrote the
```

Comment:
Actually is a general description section, which contains other info than the author.